src/Formatter/DateTimeFormatter.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Formatter;
+
+use Symfony\Component\PropertyAccess\PropertyPath;
+
+final class DateTimeFormatter implements DataFormatterInterface
+{
+    /** @var null|string */
+    private $dateTimeFormat = null;
+
+    /**
+     * DateTimeFormatter constructor.
+     * @param null|string $dateTimeFormat
+     */
+    public function __construct($dateTimeFormat = 'r')
+    {
+        $this->dateTimeFormat = $dateTimeFormat;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function supports($data)
+    {
+        return ($data instanceof \DateTime);
```

Comment:
- please remove uneeded parenthesis
- `$data instanceof \DateTime || $data instanceof \DateTimeInterface` (the left part can be dropped when dropping php < 5.5)